curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/6401db0f08030e4db8b40516.md

```diff
@@ -0,0 +1,174 @@
+---
+id: 6401db0f08030e4db8b40516
+title: Step 23
+challengeType: 0
+dashedName: step-23
+---
+
+# --description--
+
+Now, our FCC Authors Page app is complete. But we need to clean up some things.
+
+After clicking the "Load More Authors" button a couple of times, you'll see it doesn't fetch more authors anymore. That's because you've reached the end of the authors list. For a better UX, that button should be disabled at that point and the text set to something like "No more data to load". You can also set the cursor to `not-allowed` to make sure the user stop clicking it. 
+
+If you look at the authors' bios too, some are longer than the others. you can extract the part of the bios you want and replace the others with an ellipsis (...).
+
+Also, to make things look better you can add a divider line between the authors' images and every other thing in the authors cards.
+
+First, let's start with the "Load More Authors" button. When there's no more data to load, you want to change the button text and disable the button. An `if` statement would be ideal for this.
```

Comment:
We will need to remove all references to first person narrative per the [docs](https://contribute.freecodecamp.org/#/how-to-work-on-coding-challenges?id=challenge-descriptionsinstructions) here